### ISSUE TITLE
Removing dataset from plots triggers animation

### ISSUE THREAD

**Issue by jtlan** [palantir/plottable#1067]
This bug may manifest after #1002 goes in.

In `NewStylePlot`, `removeDataset()` calls `_onDatasetUpdate()`, which sets `animateOnNextRender` to **true**. This means that if we convert `Plot.Line` to a new style plot, removal of a single line will cause the others to reanimate, even though their layout is not affected by the removed line. That visual effect may be desirable though.

**Comment by aicioara** [comment]
Repro http://jsfiddle.net/cxq8L0h9/

**Comment by jtlan** [comment]
Might be desirable to retrigger animation in some cases though, such as removing a `Dataset` from `StackedBar` or `StackedArea`.

**Comment by aicioara** [comment]
@jtlan Since you requested this change, do we still want it?
Otherwise, I suggest closing as `wontfix` and wait until someone actually requests the behavior.
